apps/e2e/helpers/fixtures.ts

```diff
@@ -38,6 +53,31 @@ export const testOrders = test.extend<OrderTestFixture>({
 
 		await use(books);
 	},
+	customerOrderLines: async ({ page }, use) => {
+		await page.goto(baseURL);
+
+		const supplierOrderLine = { ...books[0], supplier_id: supplier.id, supplier_name: supplier.name, quantity: 1, line_price: 10 };
+		const dbHandle = await getDbHandle(page);
+		await dbHandle.evaluate(upsertCustomer, customer);
+
+		await dbHandle.evaluate(upsertBook, books[0]);
+		await dbHandle.evaluate(upsertBook, books[1]);
+		await dbHandle.evaluate(addBooksToCustomer, { customerId: 1, bookIsbns: [books[0].isbn, books[1].isbn] });
+		await dbHandle.evaluate(upsertSupplier, supplier);
+		await dbHandle.evaluate(associatePublisher, { supplierId: supplier.id, publisherId: "pub1" });
+		await dbHandle.evaluate(createSupplierOrder, [supplierOrderLine]);
+
+		const placedOrders = await dbHandle.evaluate(getPlacedSupplierOrders);
+
+		const placedOrderIds = placedOrders.map((placedOrder) => placedOrder.id);
+		const reconciliationOrderId = await dbHandle.evaluate(createReconciliationOrder, placedOrderIds);
+		await dbHandle.evaluate(addOrderLinesToReconciliationOrder, { id: reconciliationOrderId, newLines: [supplierOrderLine] });
+		await dbHandle.evaluate(finalizeReconciliationOrder, reconciliationOrderId);
+
+		const customerOrderLines = await dbHandle.evaluate(getCustomerOrderLineStatus, customer.id);
+
+		await use(customerOrderLines);
+	},
```

Comment:
I would much rather prefer to:
- define static data in the fixture (with ids, states, orders, etc...)
- set up the state with such static data (in DB)
- return the static data

This returns dynamic data - stores something in the DB and returns a query result. This complicates and obfuscates things (in case of debugging a failing test).

I'll merge this as I think we'll need to do a full sweep of fixtures, e2e in general and dashboard helpers, anyway.
Just leaving this here as some feedback.